virt/virtual_machines/virt-configuring-higher-vm-workload-density.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-configuring-higher-vm-workload-density"]
+= Configuring higher VM workload density
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-higher-vm-workload-density
+
+toc::[]
+
+
+:FeatureName: Configuring higher workload density
```

Comment:
No direct rule yet, but consider starting the module with a abstract sentence and then add the TP note.